libcxx/include/__algorithm/inplace_merge.h

```diff
@@ -223,14 +224,14 @@ _LIBCPP_HIDE_FROM_ABI void __inplace_merge(
 }
 
 template <class _BidirectionalIterator, class _Compare>
-inline _LIBCPP_HIDE_FROM_ABI void inplace_merge(
+_LIBCPP_HIDE_FROM_ABI inline _LIBCPP_CONSTEXPR_SINCE_CXX26 void inplace_merge(
```

Comment:
`inline` first seems to be the most common in the code base, so let's keep it like that.
```suggestion
inline _LIBCPP_HIDE_FROM_ABI  _LIBCPP_CONSTEXPR_SINCE_CXX26 void inplace_merge(
```